src/SimCan.cpp

```diff
@@ -15,10 +18,14 @@ void SimCan::begin(){
 void SimCan::handle(){
     // Check for and send CAN update
     if (millis() - _can_last_update >= CAN_UPDATE_MS) {
+        digitalWrite(_ledPin, HIGH);
```

Comment:
Not code related at all but I'm curious about whether you can actually see the LED flash? It seems like there'd be very little time before the next invocation of this loop, which would turn off the LED.

Reply:
Oh to answer this, yes you could see the LED flash, but I then realized that the LED on the Arduino board is on the same pin as the SPI clock, so there was a conflict. It's been removed.